docs/plugin_developers_guide/02_writing_plugins.rst

```diff
@@ -358,8 +358,12 @@ We have used the properties *type* and *default* to define properties of the con
      - A description of the configuration item used in the user interface to give more details of the item. Commonly used as a mouse over help prompt.
    * - displayName
      - The string to use in the user interface when presenting the configuration item. Generally a more user friendly form of the item name. Item names are referenced within the code.
+   * - items
+     - The type of the items in a list or kvlist configuration item.
```

Comment:
Explain properties For Object List type and type bucket?

For example:
 ```
type:list/kvlist
items: object
properties:{...}

OR
type: bucket
properties: {...}
```
And also explain the contract for properties...

Reply:
Buckets are not in Fledge, hence they are missing here. I had assumed we did not have support for objects yet, but if we do I will add them.

Reply:
My assumption on object was based on the fact it is not included in the definition of _valid_type_strings in configuration_manager.py. However I see it is elsewhere in that file.

Reply:
Added section on object lists

Reply:
> Buckets are not in Fledge, hence they are missing here.

Even that be tricky to be handled in white-labelled one's

Reply:
Yes, but what I really do not want to do is start putting bucket stuff in Fledge. We will need to do something, but maybe we limit it Ito a section of the documentation about storing different data types, i.e. models, metrics and MATLAB compiled scripts

Reply:
yeah, separate section can be achievable with white-labelled